exporter.go

```diff
@@ -177,11 +177,11 @@ func main() {
 
 	registry.MustRegister(buildInfoMetric)
 
+	userAgent := fmt.Sprintf("NGINX-Prometheus-Exporter/v%v", version)
+	userAgentRT := &userAgentRoundTripper{agent: userAgent, rt: &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: !*sslVerify}}}
```

Comment:
style suggestion. to improve readability, could you possibly define userAgentRT on multiple lines like this:
```go
	userAgentRT := &userAgentRoundTripper{
		agent: userAgent,
		rt: &http.Transport{
			TLSClientConfig: &tls.Config{InsecureSkipVerify: !*sslVerify},
		},
	}
```